pkg/console/metrics/register.go

```diff
@@ -1,32 +0,0 @@
-package metrics
-
-import (
-	"sync"
-
-	k8smetrics "k8s.io/component-base/metrics"
-	"k8s.io/component-base/metrics/legacyregistry"
-)
-
-type ConsoleMetrics struct {
-	ConsoleURL *k8smetrics.GaugeVec
-}
-
-var singleton *ConsoleMetrics
-var once sync.Once
-
-func Register() *ConsoleMetrics {
-	// thread safe
-	once.Do(func() {
```

Comment:
Reason to eliminate?

Reply:
I believe timing is a problem I was aiming to resolve,a lot of examples don't account for this.